src/Queue/Job.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Queue;
+
+use Symbiote\QueuedJobs\Services\AbstractQueuedJob;
+use Symbiote\QueuedJobs\Services\QueuedJob;
+
+/**
+ * Class Job
+ *
+ * This job represent a generic job which consumes items and each item is processed in one step
+ * useful as a template for jobs which need to process multiple items
+ * item can be an arbitrary piece of data for example: ID or array
+ *
+ * @property array $items
+ * @property array $remaining
+ * @package App\Queue
+ */
+abstract class Job extends AbstractQueuedJob
```

Comment:
Needs a better name :)

Reply:
Happy to rename this during standard peer request flow.